addons/mail/static/src/messaging/component/activity/activity.js

```diff
@@ -28,8 +28,9 @@ class Activity extends Component {
             areDetailsVisible: false,
         });
         useStore(props => {
+            const activity = this.env.entities.Activity.get(props.activityLocalId);
             return {
-                activity: this.env.entities.Activity.get(props.activityLocalId),
+                activity
```

Comment:
Useless diff in general but it's ok to change, easier to print the value when we need to debug, but then add `,` here